File: 2010-check-if-word-equals-summation-of-two-words/2010-check-if-word-equals-summation-of-two-words.py
class Solution(object):
    def isSumEqual(self, fw, sw, tw):
        """
        :type firstWord: str
        :type secondWord: str
        :type targetWord: str
        :rtype: bool
        """
        def change(str):
            ch = ""
            for w in str:
                curr = ord(w) - ord('a')
                ch += chr(curr + 48)
            
            return ch
        
        f = change(fw)
        s = change(sw)
        t = change(tw)
        
        return int(f) + int(s) == int(t)

File: 2010-check-if-word-equals-summation-of-two-words/2010-check-if-word-equals-summation-of-two-words.cs
public class Solution {
    public bool IsSumEqual(string firstWord, string secondWord, string targetWord) {
        int value(string str) {
            int v = 0;
            for (int i = 0; i < str.Length; ++i)
                v = v * 10 + (str[i] - 'a');
            
            return v;
        }

        return value(firstWord) + value(secondWord) == value(targetWord);
    }
}